Library/Styles/CommentCellStyles.swift

```diff
@@ -0,0 +1,61 @@
+import Prelude
+import Prelude_UIKit
+import UIKit
+
+public let commentCellRootStackViewStyle: StackViewStyle = { stackView in
+  stackView
+    |> \.axis .~ .vertical
+    |> \.layoutMargins .~ .init(topBottom: Styles.grid(3), leftRight: Styles.grid(1))
+    |> \.isLayoutMarginsRelativeArrangement .~ true
+    |> \.insetsLayoutMarginsFromSafeArea .~ false
+    |> \.spacing .~ Styles.grid(3)
+}
+
+public let commentBodyTextViewStyle: TextViewStyle = { textView in
+  textView
+    |> UITextView.lens.isScrollEnabled .~ false
+    |> UITextView.lens.textContainerInset .~ UIEdgeInsets.zero
+    |> UITextView.lens.textContainer.lineFragmentPadding .~ 0
+    |> UITextView.lens.backgroundColor .~ UIColor.ksr_white
+    |> \.textColor .~ .ksr_support_700
+    |> \.textAlignment .~ .left
+    |> \.font .~ UIFont.ksr_callout()
+    |> \.adjustsFontForContentSizeCategory .~ true
+    |> \.isEditable .~ false
+    |> \.isUserInteractionEnabled .~ false
+}
+
+public let creatorAuthorBadgeStyle: PaddingLabelStyle = { label in
+  label
+    |> \.text .~ Strings.Creator()
+    |> \.font .~ UIFont.ksr_footnote()
+    |> \.textColor .~ UIColor.ksr_create_700
+    |> \.backgroundColor .~ UIColor.ksr_create_700.withAlphaComponent(0.06)
+    |> roundedStyle(cornerRadius: Styles.grid(1))
+    |> \.adjustsFontForContentSizeCategory .~ true
+}
+
+// TODO: Internationalized in the near future.
+
+public let superbackerAuthorBadgeStyle: PaddingLabelStyle = { label in
+  label
+    |> \.text .~ localizedString(key: "Superbacker", defaultValue: "SUPERBACKER")
```

Comment:
Let's keep this value as sentence-case ("Superbacker") and convert it to uppercase with a modifier.